src/test/java/HomeWork18.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+public class HomeWork18 extends BaseTest{
+
+    @Test
+
+    public void playSongTest() throws InterruptedException {
+
+        navigateToPage();
+        provideEmail("guadalupe.medina@testpro.io");
+        providePassword("DoingitBig23!");
+        submitButton();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```